doc/man3/X509_verify_cert.pod

```diff
@@ -28,25 +32,29 @@ OpenSSL internally for certificate validation, in both the S/MIME and
 SSL/TLS code.
 
 A negative return value from X509_verify_cert() can occur if it is invoked
-incorrectly, such as with no certificate set in B<ctx>, or when it is called
-twice in succession without reinitialising B<ctx> for the second call.
+incorrectly, such as with no certificate set in I<ctx>, or when it is called
+twice in succession without reinitialising I<ctx> for the second call.
 A negative return value can also happen due to internal resource problems or if
 a retry operation is requested during internal lookups (which never happens
 with standard lookup methods).
 Applications must check for <= 0 return value on error.
 
+The X509_verify_chain() behaves like X509_verify_cert() except that its
+target certificate defaults to the first the first element of the
```

Comment:
doubled the first. Maybe also add `unless a target certificate is set` to the end of the sentence.

Reply:
Re-phrased slightly to:
```
The X509_verify_chain() behaves like X509_verify_cert() except that its
target certificate is the first element of the list of untrusted certificates
in I<ctx> unless a target certificate is set explicitly.
```